plugin-catalog/src/components/plugins/Detail.tsx

```diff
@@ -118,6 +118,21 @@ export interface PurePluginDetailProps {
   onAlertClose: () => void;
 }
 
+const pluginActionSnackbar = (closeCallback: () => void) => {
```

Comment:
Maybe this should be called `pluginSnackbarAction`?

Reply:
updated